cg/cli/workflow/commands.py

```diff
@@ -89,6 +92,29 @@ def link(context: CGConfig, case_id: str, dry_run: bool):
     analysis_api.link_fastq_files(case_id=case_id)
 
 
+@click.command("metrics-deliver")
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.pass_obj
+def metrics_deliver(context: CGConfig, case_id: str, dry_run: bool) -> None:
+    """Create and validate a metrics deliverables file for given case id.
+    If QC metrics are met it sets the status in Trailblazer to complete.
+    If failed, it sets it as failed and adds a comment with information of the failed metrics."""
+
+    analysis_api: NfAnalysisAPI = context.meta_apis[MetaApis.ANALYSIS_API]
+
+    try:
+        analysis_api.status_db.verify_case_exists(case_internal_id=case_id)
+    except CgError as error:
+        raise click.Abort() from error
+
+    analysis_api.write_metrics_deliverables(case_id=case_id, dry_run=dry_run)
+    try:
+        analysis_api.validate_qc_metrics(case_id=case_id, dry_run=dry_run)
+    except CgError as error:
+        raise click.Abort() from error
```

Comment:
Thanks for the comments 🚀  just one detail here: I moved this to `cg/cli/workflow/nf_analysis.py` based on a previous review comment since this command is only being used by the nextflow pipelines.

Reply:
Perfect 👍  I think we'll end up moving it back to commands when we migrate the cancer metrics, but for now it fits better here I agree :)